main.cpp

```diff
@@ -127,100 +173,113 @@ struct CarWash
 
 
 
-/*
-Thing 1) Coffee Corner
-5 properties:
-    1) amount of customers per day (int)
-    2) number of milk and milk substitutes (int)
-    3) number of sweeteners (int)
-    4) number of mugs (int)
-    5) price of coffee per cup (double)
-3 things it can do:
-    1) make coffee
-    2) charge customers
-    3) add flavourings to made coffee
- */
+
 
 struct CoffeeCorner
 {
-    //amount of customers per day
     int numDailyCustomer = 50;
-    //number of milk and milk substitutes
     int totalNumMilkOrSubstitutes = 7;
-    //number of sweeteners
     int numSweeteners = 4;
-    //price of coffee per cup
     double coffeeCupPrice = 7.45;
 
-    //make coffee
     void makeCoffee(int getMilkOrSubstitute, int getCoffee, bool checkNeedSweetener = false);
-    //charge customers
     double charge (double getCupPrice = 7.45);
-    //add flavourings to made coffee
     bool addFlavour(bool needsFlavour = true, int getFlavour = 4);
 };
 
-/*
-Thing 2) Theater
-5 properties:
-    1) number of ticket booths (int)
-    2) variety of snacks and drinks (double)
-    3) number of seats (int)
-    4) name of theater (std::string)
-    5) price of ticket per person (float)
-3 things it can do:
-    1) sell tickets
-    2) show movies
-    3) sell snacks and drinks
- */
+void CoffeeCorner::makeCoffee(int getMilkOrSubstitute, int getCoffee, bool checkNeedSweetener)
+{
+    if (checkNeedSweetener == true)
+    {
+        numSweeteners -= 1;
+    }
+
+    getMilkOrSubstitute = totalNumMilkOrSubstitutes;
+    getCoffee = 1;
+    std::cout << "Coffee Ready!\n" << "Coffee Type: " << totalNumMilkOrSubstitutes << " X" << getCoffee << "\n";
+}
+
+double CoffeeCorner::charge(double getCupPrice)
+{
+    int totalNumCups = 2;
+    getCupPrice = coffeeCupPrice;
+    double totalCharge = totalNumCups * getCupPrice;
+    std::cout << "Your charge is: " << totalCharge;
+    return totalCharge;
+}
+
+bool CoffeeCorner::addFlavour(bool needsFlavour, int getFlavour)
+{
+    if (needsFlavour == true)
+    {
+        getFlavour = 2;
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
```

Comment:
If you find yourself writing code like this:
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    else 
    {
        return anotherValue;
    }
}
```
You do not need the `else` block.  Here's why:
`return` statements are the last thing that **ever happens** in a function.
When the `if()` expression is evaluated as `true` and the block is entered, the `return aValue;` is the last line of code to be executed before the function exits.
but when the `if()` expression evaluates as `false` and the block is **NOT** entered, the next thing that happens is `return anotherValue;`
you have no code after the `else` block that needs executing, since your `else` block contains a `return` statement.
that means you do **NOT** need the `else` block, and it is just causing visual clutter.
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    
    return anotherValue;
}
```
- [ ] revise